test/run-stp-tests.sh

```diff
@@ -1025,7 +1029,15 @@ else
 fi
 
 # Error log messages
-count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | \
+       $GREP -v '(usb) crashed on signal 11' | \
+       $GREP -v '(dnssd) stopped with status 1' | \
+       $GREP -v 'loadFile failed: temp file: not a PDF file' | \
+       $GREP -v 'Failed to connect to system bus' | \
+       $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | \
+       $GREP -v 'Unable to write uncompressed print data: Broken pipe' | \
+       $GREP -v 'Job held by' | \
```

Comment:
Where was *this* showing up?

Reply:
So I removed this filtering-out and uploaded. This appeared on [`mips64el`](https://buildd.debian.org/status/fetch.php?pkg=cups&arch=mips64el&ver=2.3.3op1%7E89-g26e085bfd-2&stamp=1605290790&raw=0) and [`mipsel`](https://buildd.debian.org/status/fetch.php?pkg=cups&arch=mipsel&ver=2.3.3op1%7E89-g26e085bfd-2&stamp=1605291505&raw=0) architectures (the links point to build-logs).

I can't tell whether it's an architecture-specific problem or a "slowness" problem. This error-line counting is the only issue in the test-suite, so I leaned towards the latter. Opinions?

Reply:
@OdyX It looks like the "Job held by" message is the culprit, which was changed for [Issue #5604](https://github.com/apple/cups/issues/5604) but the fix I made has side-effects so I'm going to back that out and make the "auth required" backend status log its own message instead.

(which is what this automated test suite is supposed to catch :)

Reply:
[master 2a868d14c] Fix a logging regression that was detected by the automated test suite (Issue #25)

Reply:
[master 1476b03a4] Don't log errors when IPv6 listeners fail because IPv6 is disabled (Issue #25)